app/models/query.rb

```diff
@@ -41,4 +43,8 @@ def self.to_csv(*_args)
       end
     end
   end
+
+  def remove_from_search_api
+    SearchApiSaver.new(self).destroy(action: :delete)
```

Comment:
Oddly, `SearchApiSaver.new(self).destroy` will also destroy the object, so I wonder whether we should just override destroy on this model?

```ruby
  def destroy(action: :delete)
    ActiveRecord::Base.transaction do
      @object.destroy!
```

Reply:
How about 721e557c74f6de61809d1c6a997377deec7d3158?

Reply:
Looks good 👍